### DISCUSSION TITLE
Ability to generate "offline" provisioning profiles

### DISCUSSION THREAD

**Discussion by ZonD80** [fastlane/fastlane#19295] [Ideas]
Apple has introduced new "offline" provisioning profiles that are valid for 7 days only, here how it looks like in dev center (valid for newer accounts (I can see it in recently created clients account (~5th of August), but I can't see this on our company account (around 2 years already)):
<img width="1080" alt="image" src="https://user-images.githubusercontent.com/273057/130915210-e8a25444-a33b-4306-bf31-de13710a12ec.png">

It will be great if we can generate such provisioning with fastlane as well

**Comment by fedulvtubudul** [comment]
Same problem for me. Have several developer profiles to generate which I have to do each week. Really need this!

**Comment by tomieq** [comment]
When can we expect some improvements in this area as it would be nice feature?
